_locales/cs/maqueenPlusV2-jsdoc-strings.json

```diff
@@ -1,23 +1,52 @@
 {
     "maqueenPlusV2.controlMotor": "Blok pro ovládání motorů. Umožňuje měnit směr a rychlost otáček.",
-    "maqueenPlusV2.controlLED": "Blok pro ovládání předních LED světel.",
+    "maqueenPlusV2.controlMotor|param|emotor": "Identifikátor motoru z MyEnumMotor, např.: MyEnumMotor.LeftMotor",
+    "maqueenPlusV2.controlMotor|param|edir": "Směr otáček motoru z MyEnumDir, např.: MyEnumDir.Forward",
+    "maqueenPlusV2.controlMotor|param|speed": "Rychlost otáček motoru v rozsahu od 0 do 255, např.: 100",
+    "maqueenPlusV2.controlLED": "Blok pro ovládání předních LED světel (reflektorů).",
+    "maqueenPlusV2.controlLED|param|eled": "Identifikátor LED z MyEnumLed, např.: MyEnumLed.LeftLed",
+    "maqueenPlusV2.controlLED|param|eSwitch": "Nový stav LED z MyEnumSwitch, např.: MyEnumSwitch.Close",
     "maqueenPlusV2.readLineSensorState": "Vrátí hodnotu 1, pokud vybraný senzor detekuje čáru, jinak vrátí 0.",
+    "maqueenPlusV2.readLineSensorState|param|eline": "Identifikátor senzoru čáry z MyEnumLineSensor, např.: MyEnumLineSensor.SensorL1",
     "maqueenPlusV2.readLineSensorData": "Vrátí analogovou hodnotu (celé číslo) z vybraného senzoru sledování čáry.",
+    "maqueenPlusV2.readLineSensorData|param|eline": "Identifikátor senzoru čáry z MyEnumLineSensor, např.: MyEnumLineSensor.SensorL1",
     "maqueenPlusV2.readUltrasonic": "Vrátí vzdálenost robota od překážky v cm. Ultrazvukový senzor ma dosah 2 až 500 cm a v případě chyby vrací hodnotu 0.",
-    "maqueenPlusV2.readUltrasonic|param|echo": "Pin pro čtení hodnoty signálu ECHO. Např. : DigitalPin.P14",
-    "maqueenPlusV2.readUltrasonic|param|trig": "Pin pro zápis signálu TRIG. Např. : DigitalPin.P13",
+    "maqueenPlusV2.readUltrasonic|param|echo": "Pin pro čtení hodnoty signálu ECHO, např.: DigitalPin.P14",
+    "maqueenPlusV2.readUltrasonic|param|trig": "Pin pro zápis signálu TRIG, např.: DigitalPin.P13",
```

Comment:
Not sure what TRIG/ECHO are about. Might be a good idea to add a short explanation in brackets.